### PR TITLE
Zephyr SDK Version Bump v0.17.2->v0.17.3

### PR DESCRIPTION
**Pull request by BujSet**
### Summary
Bumping Zephyr SDK version to v0.17.3, similar to #13380.

### Test plan
Same as #13380, verified correct output on support models (`add`, `softmax`, and `mv2`). 

cc @psiddh @AdrianLundell @digantdesai

### CODE REVIEW

**Review by digantdesai — APPROVED** [review]
@psiddh do you want to take this over?

**Comment by psiddh** [comment]
> @psiddh do you want to take this over?

@digantdesai  Sure. Just rebased the changes, hopefully CI is ok

**Review by psiddh — APPROVED** [review]

**Comment by zingo** [comment]
I have a change (ongoing not a PR yet) bumbing to v0.17.4 I also see v1.0.0 Beta is relesed but I assume (and think) we can wait for the proper release and bump to it then.

**Comment by zingo** [comment]
Replaced by https://github.com/pytorch/executorch/pull/16539
